### PR TITLE
Initial merge request to get watch to usable state

### PR DESCRIPTION
**Pull request by dodoradio**
Currently, the interface of the watch is usable.
Hands are put into timekeeping mode by init.rc but are not synchronised to system time
all other peripherals are not working, and the watch seems to be having issues with suspend.

### CODE REVIEW

**Comment by dodoradio** [comment]
there do seem to still be some orphaned files relating to firefish, as this is based on the ray repo. Please flag this up if you want me to clean this up.

File: README-system-dir.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # /system/ directory build instructions
```
[the remark this answers is not in the document]
Comment (dodoradio) [reply README-system-dir.md]: I have trimmed this file to only contain information about getting to fastboot menu and have updated with images of narwhal.

File: recipes-android/android-init/android-init/init.rc [hunk 2]
```diff
@@ -44,11 +46,11 @@ service config_bt_addr /vendor/bin/btnvtool -O
     class core
     oneshot
 
-service bt /vendor/bin/android.hardware.bluetooth@1.0-service-qti
+service bt /vendor/bin/hw/android.hardware.bluetooth@1.0-service
     class core
     oneshot
 
 service wcnss_service /vendor/bin/wcnss_service
```
[the remark this answers is not in the document]
Comment (dodoradio) [reply recipes-android/android-init/android-init/init.rc:46]: removed this

File: recipes-android/android/android_narwhal-n.bb [hunk 3]
```diff
@@ -15,7 +15,7 @@ PV = "oreo"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
 INHIBIT_PACKAGE_STRIP = "1"
-COMPATIBLE_MACHINE = "ray|firefish"
+COMPATIBLE_MACHINE = "narwhal|firefish"
```
[the remark this answers is not in the document]
Comment (dodoradio) [reply recipes-android/android/android_narwhal-n.bb]: removed all references to firefish in initial rename commit

File: recipes-android/android/android_narwhal-n.bb [hunk 4]
```diff
@@ -4,7 +4,7 @@ SUMMARY = "Downloads the Snapdragon Wear 2100/3100 /usr/libexec/hal-droid and /u
 LICENSE = "CLOSED"
 
 SRC_URI = "https://dl.dropboxusercontent.com/s/yl8tezw6e2d661o/hybris-o-msm8909.tar.gz;name=hybris \
-    https://dl.dropboxusercontent.com/s/al40eklhv50l6z5/system-ray-pie-preview.tar.gz;name=system \
+    https://dodorad.io/system-narwhal-pie-preview.tar.gz;name=system \
```
[the remark this answers is not in the document]
Comment (dodoradio) [reply recipes-android/android/android_narwhal-n.bb]: updated the link

**Comment by dodoradio** [comment]
Cleaned up to remove firefish, reworked history and disabled bluetooth service

File: conf/machine/narwhal.conf [hunk 5]
```diff
@@ -12,7 +12,8 @@ MACHINE_HAS_WLAN = "true"
 PREFERRED_PROVIDER_virtual/android-system-image = "android"
 PREFERRED_VERSION_android = "oreo"
 
-PREFERRED_PROVIDER_virtual/kernel = "linux-firefish"
+PREFERRED_PROVIDER_virtual/kernel = "linux-narwhal"
 PREFERRED_VERSION_linux = "3.18+oreo"
 
-IMAGE_INSTALL += "underclock udev-droid-system wifi-enabler bluebinder asteroid-hrm"
+IMAGE_INSTALL += "underclock udev-droid-system wifi-enabler bluebinder asteroid-compass"
```
Comment (MagneFire) [inline conf/machine/narwhal.conf:18]: We don't need `wifi-enabled` and `bluebinder` at this point.
I would assume that your journal is now being spammed with `bluebinder` related messages?

**Review by MagneFire — COMMENTED** [review]
Only found the one final thing (which I should've seen before, sorry about that).
Otherwise looks good to me now.

**Comment by FlorentRevest** [comment]
I'd just add, please make the images under `images/` much smaller (they currently take 10MB each)

... or completely remove them if you want. I don't think this is necessary to have these and even less so in layers. These would get cloned every time someone wants to build asteroid, it's not really necessary. Also once we introduce a blob in a git log then it stays there forever after. So if you choose to resize or remove them, make sure you edit the commit that introduces them otherwise if you keep a commit that introduces it and a later one that removes it, it will still be cloned as part of the git history in the future.

Otherwise, the rest looks great, thanks a lot!

**Pull request merged**